packages/sdk/src/models/PrimerSettings.ts

```diff
@@ -98,7 +98,6 @@ interface IPrimerApplePayOptions {
    * @deprecated Use Client Session API to provide merchant name value: https://primer.io/docs/payment-methods/apple-pay/direct-integration#prepare-the-client-session
    */
   merchantName?: string;
-  isCaptureBillingAddressEnabled?: boolean;
```

Comment:
[question]: should we keep and deprecate this?

Reply:
Nice catch, it should be deprecated and live under `IPrimerApplePayOptions` not `IBillingOptions`. Thanks! Pushed the update